### PR TITLE
Tenant deployment test

### PR DESCRIPTION
**Pull request by cniackz**
The idea is to test minio operator deployment on every commit, making sure we are
not breaking it after the changes.

### CODE REVIEW

File: testing/basic-tenant.sh [hunk 1]
```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+# Copyright (C) YYYY, MinIO, Inc.
```
Comment (harshavardhana) [inline testing/basic-tenant.sh:3]:
```suggestion
# Copyright (C) 2022, MinIO, Inc.
```

File: testing/basic-tenant.sh [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+echo "Installing lite tenant"
+
+try kubectl apply -k ../examples/kustomization/tenant-lite
+
+
+echo "Waiting for the tenant statefulset, this indicates the tenant is being fulfilled"
+waitdone=0
+totalwait=0
+while true; do
+  waitdone=$(kubectl -n tenant-lite get pods -o json | jq '.items | length')
```
Comment (harshavardhana) [inline testing/basic-tenant.sh:51]:
```suggestion
  waitdone=$(kubectl -n tenant-lite get pods -l v1.min.io/tenant=storage-lite --no-headers | wc -l)
```

File: testing/basic-tenant.sh [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License, version 3,
+# along with this program.  If not, see <http://www.gnu.org/licenses/>
+
+# This script requires: kubectl, kind, jq
```
Comment (harshavardhana) [inline testing/basic-tenant.sh:17]:
```suggestion
# This script requires: kubectl, kind
```

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: testing/basic-tenant.sh (hunk 1 above)
Comment (cniackz) [reply testing/basic-tenant.sh]: Done the year got changed. Thanks :)

**Comment by cniackz** [comment]
I will close this pull request and keep Daniel's pull request only for this change.

**Pull request closed without merging**